graph_generation/generate_from_config.py

```diff
@@ -0,0 +1,230 @@
+"""
+This module generates a random data-dependancy mapping graph from config file.
+You can create graph in proto format, or in networkx format.
+
+Usage:
+    python3 graph_generation/generate_from_config.py \
+         --output_file "output.graphml" \
+         --config_file "graph_generation/configs/config_15_09_20.yaml" \
+         --graph_type "networkx" \
```

Comment:
Please also specify here what are the possible values for graph_type and what is the effect of overwrite (what happens if overwrite is not set and the file exists?)

Reply:
This information is shown whenever the module is used and help will explain all the default values for arguments (for overwrite the default is False, so no one loses data unintentionally). I can add this info to module documentation as well if it will help to clear out the usage.